scene/gui/menu_bar.cpp

```diff
@@ -418,6 +424,7 @@ Rect2 MenuBar::_get_menu_item_rect(int p_index) const {
 	ERR_FAIL_INDEX_V(p_index, menu_cache.size(), Rect2());
 
 	Ref<StyleBox> style = theme_cache.normal;
+	bool rtl = is_layout_rtl();
```

Comment:
This can probably be declared right above the conditional check, since it's the only place that uses it. Also, could this variable be marked as const? (Same above, if it could).